### PR TITLE
Ft 47 chat window styling

### PR DESCRIPTION
**Pull request by LaurinUB**
- add styling for chat window
- add moveable window component

### CODE REVIEW

File: frontend/src/components/moveablewindow/Moveablewindow.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from "react";
```
Comment (philippschwarzmueller) [inline frontend/src/components/moveablewindow/Moveablewindow.tsx:1]: useEffect is unused

File: frontend/src/components/moveablewindow/Moveablewindow.tsx
Comment (philippschwarzmueller) [inline frontend/src/components/moveablewindow/Moveablewindow.tsx:1]: file is not formatted according to prettier

File: frontend/src/components/moveablewindow/Moveablewindow.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  return (
+    <>
+      <StyledWindow style={{top: position.y, left: position.x}}>
+      <Windowbar
+      draggable={true}
+      onDragStart={startDrag}
+      onDrag={handleDrag}
+      onDragEnd={endDrag}
+      onDragOver={handleDrop}
+      ><img width="16" height="16" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAABQ0lEQVR4AcXBAW6DMBBFwedo78XezH9vtj6ZS5pURSgECoTOFN6Q1DmZpMKEsUBSd3fOMgwDEcGoA4WnGy9I6u7OWYZhICJ4xZiR1GutnCkiWGJMSOq1VlprXMVYkJkcVWultcY7xgJJHCNqZZXxVme/YIsbazofZWxWWNbZy1hT+CaJWitzpRSOMHYoJfglIHgQ0PkLYwcJaq3MlSL+ythIEpK4k8RZjM06v4KzGDtIQhJnMHbpnMVYJCD4NGNB75UrGAtaa5xNEpIYdUaSijETEXyCJCRx5+5kJqNuTEjiUyRx5+5kJj+MkaTOBdydzGTKJPVaK1eICOaMp9Ya/8E4IDPZwt1ZYsz44NxlS9ZIYovMZIkxFzw4qzKTVzITd2cLYyY9OcrdWSOJUbnxDyQxKoyMUURwFUmMCk+Fh851ChNfhxx7+xF1KZkAAAAASUVORK5CYII="></img>
```
Comment (philippschwarzmueller) [inline frontend/src/components/moveablewindow/Moveablewindow.tsx:89]: img elements need an alt attribute (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img)

File: frontend/src/components/nav/Navbar.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ const StyledNavbar = styled.nav`
   height: 5vh;
   z-index: 2;
   background-color: rgb(195, 199, 203);
+  /* background-color: LightGray; */
```
Comment (philippschwarzmueller) [inline frontend/src/components/nav/Navbar.tsx:14]: is this rly neccessary?

File: frontend/src/components/chatwindow/Chatwindow.tsx
Comment (philippschwarzmueller) [inline frontend/src/components/chatwindow/Chatwindow.tsx:1]: file not formatted according to prettier

**Review by philippschwarzmueller — CHANGES_REQUESTED** [review]
some easy and quick changes needed (see comments)
THX for the awesome contribution !

**Review by LaurinUB — COMMENTED** [review]
will be fixed

File: frontend/src/components/nav/Navbar.tsx (hunk 3 above)
Comment (LaurinUB) [reply frontend/src/components/nav/Navbar.tsx]: no forgot i put it for color checks

**Review by philippschwarzmueller — APPROVED** [review]
:shipit:

**Pull request merged**